src/components/RangeSlider/components/DualThumb/DualThumb.tsx

```diff
@@ -197,7 +197,10 @@ export default class DualThumb extends React.Component<Props, State> {
         >
           <div className={styles.Wrapper} id={id}>
             {prefixMarkup}
-            <div className={trackWrapperClassName}>
+            <div
+              className={trackWrapperClassName}
+              onClick={this.handleTrackClick}
```

Comment:
We’ll want this to be a `onMouseDown` to follow the [UI Events spec](https://www.w3.org/TR/uievents/) as it will be the start of a drag event. A mouse down and hold should permit dragging the respective thumb.

Reply:
Even if this isn't a dragging motion? The intention is to be able to click and have the thumb jump to the clicked position.

![click-track](https://user-images.githubusercontent.com/6239480/52502626-16db3980-2b98-11e9-9c6a-18cf98054c21.gif)